.gitignore

```diff
@@ -1,11 +1,17 @@
-*.iml
+
 *.log
 *.swp
 .classpath
 .gradle/
-.idea/
 .project
 .settings/
 /bin
 bin/
-build/
+build
```

Comment:
Why this change? We want to ignore the build directory but not the file build if any under the project root, right?